core/src/components/alert/alert.scss

```diff
@@ -152,6 +152,8 @@
 
   width: 100%;
   height: 100%;
+
+  min-height: inherit;
```

Comment:
Without this the button inner may have visual diffs since the height is no longer fixed at 48px